alf/utils/common.py

```diff
@@ -878,6 +893,24 @@ def get_conv_layers(conv_layer_params):
     return image_processing_layers
 
 
+@gin.configurable
+def get_identity_layer(input_shape, batch_size):
+    """Generate an Identity Layer of input_shape and batch_size.
+
+    Args:
+      input_shape (list of ints): input tensor shape.
+      batch_size (int): batch size.
+
+    Returns:
+      A network that passes input directly to output.
+    """
+    x = tf.keras.Sequential()
+    x.add(
+        tf.keras.layers.Lambda(
+            lambda x: x + 0, input_shape=input_shape, batch_size=batch_size))
```

Comment:
why do you need to pass input_shape and batch_size? They are not the required arguments for Lambda()